tests/qnn/test_keras.py

```diff
@@ -418,6 +418,27 @@ def c_default(w1, w2, w3, w4, w5, w6, w7, inputs=None):
         assert layer_out.shape == (batch_size, output_dim)
         assert np.allclose(layer_out[0], c(x[0], *weights))
 
+    @pytest.mark.parametrize("n_qubits, output_dim", indices_up_to(1))
```

Comment:
How about testing for multiple qubits too? Also, we could maybe have 1-2 more values for `batch_size` and `middle_dim` (parameterization was already nicely set up :slightly_smiling_face: ). This latter point applies in the Torch case too.

Reply:
Sure, sounds good. I added a couple extra values to each parameter by just picking a couple of small numbers, is there an established convention for choosing parameters for tests you'd prefer I followed?

Reply:
No, this will be great! Thanks!